crates/bevy_window/src/window.rs

```diff
@@ -314,6 +314,80 @@ pub struct Window {
     ///
     /// - Only used on iOS.
     pub recognize_pan_gesture: Option<(u8, u8)>,
+    /// Enables click-and-drag behavior for the entire window, not just the titlebar.
+    ///
+    /// Corresponds to [`WindowAttributesExtMacOS::with_movable_by_window_background`].
+    ///
+    /// # Platform-specific
+    ///
+    /// - Only used on macOS.
+    ///
+    /// [`WindowAttributesExtMacOS::with_movable_by_window_background`]: https://docs.rs/winit/latest/x86_64-apple-darwin/winit/platform/macos/trait.WindowAttributesExtMacOS.html#tymethod.with_movable_by_window_background
+    pub movable_by_window_background: bool,
+    /// Makes the window content appear behind the titlebar.
+    ///
+    /// Corresponds to [`WindowAttributesExtMacOS::with_fullsize_content_view`].
+    ///
+    /// For games which want to render the window buttons on top of the game
+    /// itself, this should be enabled along with [`titlebar_transparent`].
```

Comment:
Nit: Bevy isn't just used for games :) 
```suggestion
    /// For apps which want to render the window buttons on top of the app
    /// itself, this should be enabled along with [`titlebar_transparent`].
```

Reply:
Good catch!